### ISSUE TITLE
Serious accessibility problem in live chat tab

### ISSUE THREAD

**Issue by kastwey** [microsoft/BotFramework-Emulator#2076]
Hi,

I'm blind, and I'm using a screen reader.
After downloading the latest version (I updated today), the list of messages are not visible for JAWS screen reader.
The error, I think, is because you put aria-label="" in the span containing the message:
<div class="webchat__row message"><span aria-label=" " class="css-1i0rgxs">Bot  said, What else can I do for you?.
So the span content is not shown because the label is empty.
Could you fix this, please? I'm developing a bot but I cannot use the emulator due this accessibility issue.

On the other hand, in the chat panel, you could use aria-live to mark that panel as a live area region, and have the screen reader automatically read the new messages. It would be great!

Thanks in advance.

Kind regards,

Juanjo.

**Comment by EricDahlvang** [comment]
Hi @kastwey or Juanjo,

I've worked with JAWS, nearly 20 years ago. It is good to hear the application is still prominent and useful.

Thank you for raising this issue and recommendations.  The team will address it asap.

**Comment by kastwey** [comment]
Hi!

Thanks so much, really!

All the best,

Juanjo.


2020-02-09 6:24 GMT+01:00, Eric Dahlvang <notifications@github.com>:

**Comment by clearab** [comment]
@cwhitten Can you take a look at this and prioritize the bug fix?

**Comment by cwhitten** [comment]
@kastwey thank you for the issue and I’m sorry to hear you’re having screenreader issues using the Emulator.

We recently completed WCAG 2.1 compliance criteria, but the audit was done using NVDA and not JAWS. This is due to the underlying browser and web technologies that power the application. Are you able to use NVDA instead?

**Comment by kastwey** [comment]
Hi!

Yes, NVDA does read messages from the logs, but when new messages
arrive it doesn't read them automatically.

I think what with JAWS,  the conflict is due the empty aria-label
inside the message container. NVDA may overlook it, but JAWS doesn't.

Thanks!

Best regards,

Juanjo.

2020-02-11 4:33 GMT+01:00, Chris Whitten <notifications@github.com>:

**Comment by cwhitten** [comment]
Hi @kastwey. Unfortunately, we don't officially support JAWS and ask our users to use NVDA with the Bot Framework Emulator. 

Regarding the new messages not being read immediately, can you create a new issue? We'll triage and address it as soon as we can.

**Comment by corinagum** [comment]
closing due to inactivity
